ganghee/MyStudy2/app/src/main/java/com/example/mystudy/ui/UpbitFragment.kt

```diff
@@ -13,14 +13,15 @@ import com.example.mystudy.R
 import com.example.mystudy.adapter.TickerAdapter
 import com.example.mystudy.data.FormatTickers
 import com.example.mystudy.data.UpbitRepository
-import kotlinx.android.synthetic.main.fragment_upbit.*
+import com.example.mystudy.databinding.FragmentUpbitBinding
 import org.jetbrains.anko.support.v4.toast
 
 class UpbitFragment : Fragment(), UpbitContract.View {
 
     private lateinit var tickerAdapter: TickerAdapter
     override lateinit var presenter: UpbitContract.Presenter
     private val MARKET_NAME = "market name"
```

Comment:
석준님 리뷰중에
```suggestion
   const val MARKET_NAME = "market name"
```
이렇게 상수로 companion object로 관리하라는 리뷰를 봤던거같아요!

Reply:
네! 수정하였습니다!!
[UpbitFragment](https://github.com/StudyFork/GoogryAndroidArchitectureStudy/pull/195/commits/74cc9e99b83e10f34ab1bbf083893da820e78a62)